### PR TITLE
Added timer for Demo that changes props every 0.2 sec

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
Added timer for the demo that changes progress every 0.2 sec imitating playback.

@aol-public/mobile-sdk-team ready for review.

[JIRA Issue](https://jira.ouroath.com/browse/OMSDK-796)

### CODE REVIEW

File: Demo/Demo/AppDelegate.swift [hunk 1]
```diff
@@ -5,6 +5,8 @@ import UIKit
 import PlayerControls
 
 typealias Props = ContentControlsViewController.Props
+fileprivate var progress: ContentControlsViewController.Props.Progress = 0
+
 func props() -> Props {
```
Comment (AndriiMoskvin) [inline Demo/Demo/AppDelegate.swift]: Why not just pass progress to the function?

Comment (RomanTysiachnik) [reply Demo/Demo/AppDelegate.swift]: @AndriiMoskvin fixed

File: Demo/Demo/AppDelegate.swift [hunk 2]
```diff
@@ -5,7 +5,9 @@ import UIKit
 import PlayerControls
 
 typealias Props = ContentControlsViewController.Props
-func props() -> Props {
+
+func props(progress: ContentControlsViewController.Props.Progress? = nil) -> Props {
+    let progress = progress ?? 1
```
Comment (BogdanBilonog) [inline Demo/Demo/AppDelegate.swift]: Why not defined in func param?

Comment (AndriiMoskvin) [reply Demo/Demo/AppDelegate.swift]: Why it is optional?

Comment (RomanTysiachnik) [reply Demo/Demo/AppDelegate.swift]: @BogdanBilonog @AndriiMoskvin I've removed optional. I'm agreed, that it is not really necessary.

File: Demo/Demo/AppDelegate.swift [hunk 3]
```diff
@@ -123,5 +135,3 @@ func sideProps() -> [SideBarView.ButtonProps]{
     
     return [later, favorite, share, add]
 }
-
```
Comment (BogdanBilonog) [inline Demo/Demo/AppDelegate.swift]: Leave last line empty

Comment (RomanTysiachnik) [reply Demo/Demo/AppDelegate.swift]: @BogdanBilonog all is done.

**Pull request merged**